File: _assets/scripts/normalizers/render_controllers_fix_old.lua
return function (data)
    local data_attr = python.as_attrgetter(data)
    if data_attr.__contains__("render_controllers") then
        return
    end
    local output = python.builtins.dict()
    output["defined_in"] = data["defined_in"]
    output["render_controllers"] = data
    data_attr.__delitem__("defined_in")
    return output
end

File: _assets/scripts/normalizers/render_controllers_fix_old.py
from typing import Any

__all__ = ["render_controllers_fix_old"]

def render_controllers_fix_old(data:Any) -> Any|None:
    if "render_controllers" in data:
        return None
    output = {
        "defined_in": data["defined_in"],
        "render_controllers": data,
    }
    del data["defined_in"]
    return output
